Velocity_Model/scripts/readme.md

```diff
@@ -0,0 +1,114 @@
+# Preparation for Vs30map update
```

Comment:
I reckon put all of these scripts that pertain to this process in a subfolder.

Reply:
Sure - will make a separate subfolder.